library/commandline/src/modules/CommandLine.rb

```diff
@@ -360,7 +360,7 @@ def Parse(arguments)
         # translators: error message in command line interface
         Error(Builtins.sformat(_("Unknown Command: %1"), command))
 
-        return { "command" => command }
+        return {}
```

Comment:
This fix works because our caller actually obeys our API documented above

```rb
#  @return [Hash{String => Object}]  containing the command and it's option. In case of
#        error it is an empty map.
```